mesonbuild/dependencies/ui.py

```diff
@@ -189,7 +189,7 @@ def __init__(self, name: str, environment: 'Environment', kwargs: T.Dict[str, T.
         super().__init__(name, environment, kwargs, language=language)
 
         try:
-            self.vulkan_sdk = os.environ['VULKAN_SDK']
+            self.vulkan_sdk = os.getenv('VULKAN_SDK', os.environ['VK_SDK_PATH'])
```

Comment:
we usually use `os.environ.get`, not `os.getenv`, otherwise this looks fine to me.

Reply:
done